test/mitmproxy/net/test_zstd.py

```diff
@@ -0,0 +1,21 @@
+from mitmproxy.net.encoding import decode_zstd
```

Comment:
Please add this to `test_encoding.py`. We want to keep parity between source file and test file name. :)

Reply:
Sure! Thanks for the guidance.

Reply:
Moved the test to test_encoding.py.